python-controller/inkkeys/protocol.py

```diff
@@ -19,7 +19,7 @@ class CommandCode(Enum):
     INFO = "I"
 
 class RefreshTypeCode(Enum):
-    PARTIAL = "p"
+    PARTIAL = "f"
```

Comment:
This was the line that made the screen to work. For some reazon the partial refresh is not working with the v2 board, so this can be a temporary fix until we find a proper soltuion with a library update or some firmware refactor